src/filters.rs

```diff
@@ -60,6 +60,46 @@ pub fn upcase(input: &Value, _args: &[Value]) -> FilterResult {
     }
 }
 
+
+pub fn downcase(input: &Value, _args: &[Value]) -> FilterResult {
+    match *input {
+        Str(ref s) => Ok(Str(s.to_lowercase())),
+        _ => Err(InvalidType("String expected".to_owned())),
+    }
+}
+
+
+pub fn capitalize(input: &Value, _args: &[Value]) -> FilterResult {
+    match *input {
+        Str(ref s) => Ok(Str(s.char_indices().fold(String::new(), |word, (_, chr)| {
+            let next_char = match word.chars().last() {
+                Some(last) =>
+                    if last.is_whitespace() {
+                        chr.to_uppercase().next().unwrap()
+                    } else {
+                        chr
+                    },
+                _ => chr.to_uppercase().next().unwrap(),
+            }.to_string();
+            word + &next_char
```

Comment:
Some random suggestions:

Converting every `char` to a `String` and then appending that to the other string is isn't the best way to do this. You should probably use `String::push` instead. Btw. `word` as the variable name for the accumulator string is a bit misleading, I think.

You're also iterating over the accumulated string to get the last char in every iteration, which means the whole function is O(n^2) in the length of the input string. That's not necessarily a problem, but it would be easy to avoid by just having a boolean that keeps track whether the last char was whitespace. That would make the function O(n) and also make it easier to read.

Finally, you're iterating over `char_indices`, but never actually look at the indices. Is there a reason you couldn't just use `chars` instead?